src/spark/utils.py

```diff
@@ -58,32 +63,39 @@ def _stop_spark_session(spark):
     spark.stop()
 
 
-def get_base_spark_conf(app_name: str) -> SparkConf:
+def get_base_spark_conf(
```

Comment:
Should this be a private method? If not executor_cores should probably be a default arg

Reply:
👍 i think it makes sense to make it a private method.